runtime/util/util.cpp

```diff
@@ -28,4 +28,13 @@ block *constructKItemInj(void *subject, const char *sort, bool raw_value) {
   auto args = std::vector<void *>{add_indirection ? (void *)&subject : subject};
   return static_cast<block *>(constructCompositePattern(tag, args));
 }
+
+void printProofHintHeader(char *output_file) {
+  unsigned int version = 1;
```

Comment:
https://github.com/runtimeverification/llvm-backend/blob/master/include/kllvm/binary/version.h maybe pull this class upwards out of the `binary` library and reuse it here?